src/golang/lib/execution_environment/python_version.go

```diff
@@ -0,0 +1,24 @@
+package execution_environment
+
+import (
+	"os/exec"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// GetServerPythonVersion performs a best-effort fetch of the server's python version. This should only be used when conda is not available.
+func GetServerPythonVersion() string {
```

Comment:
How about just `func PythonVersion()` ? No strong preference

Reply:
yeah I also don't have a strong preference, but I was told that `Get()` implies that we're doing more processing before deriving the returned data, which is the case here. I think I slightly prefer the current form though.